intro-to-R-tidyverse/01-intro_to_base_R.Rmd

```diff
@@ -1,29 +1,32 @@
 ---
 title: "Introduction to R and RStudio"
-author: "Originally authored by Stephanie J. Spielman,<br>adapted for the CCDL workshop by Candace Savonen"
-date: "2020"
+author: Originally authored by Stephanie J. Spielman,<br>adapted by CCDL for ALSF
+date: '2021'
```

Comment:
Just for consistency... We don't really need quotes anywhere in the header, but somehow this one jumped out at me.
```suggestion
date: 2021
```